e2e/specs/mattermost/copy_link.test.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+'use strict';
+
+const fs = require('fs');
+
+const {clipboard} = require('electron');
+
+const env = require('../../modules/environment');
+const {asyncSleep} = require('../../modules/utils');
+
+describe('copylink', function desc() {
+    this.timeout(40000);
+
+    const config = env.demoMattermostConfig;
+
+    beforeEach(async () => {
+        env.cleanDataDir();
+        env.createTestUserDataDir();
+        env.cleanTestConfig();
+        fs.writeFileSync(env.configFilePath, JSON.stringify(config));
+        await asyncSleep(1000);
+        this.app = await env.getApp();
+        this.serverMap = await env.getServerMap(this.app);
+    });
+
+    afterEach(async () => {
+        if (this.app) {
+            await this.app.close();
+        }
+    });
+
+    it('MM-T125 Copy Link can be used from channel LHS', async () => {
+        const loadingScreen = this.app.windows().find((window) => window.url().includes('loadingScreen'));
+        await loadingScreen.waitForSelector('.LoadingScreen', {state: 'hidden'});
+        const firstServer = this.serverMap[`${config.teams[0].name}___TAB_MESSAGING`].win;
+        await env.loginToMattermost(firstServer);
+        await firstServer.waitForSelector('#sidebarItem_off-topic');
+        await firstServer.click('#sidebarItem_off-topic');
+        await firstServer.click('#sidebarItem_off-topic', {button: 'right'});
+        await firstServer.click('div[role="menuitem"]:has-text("Copy Link")');
+        await firstServer.click('#sidebarItem_town-square');
+        await firstServer.click('#post_textbox');
+        const clipboardText = clipboard.readText();
+        await firstServer.fill('#post_textbox', clipboardText);
+        const content = await firstServer.locator('#post_textbox').textContent();
+        content.should.be.equal('/developers/channels/off-topic');
```

Comment:
Just noticed this, the `developers` team won't exist on the test server either, you will have to use the team provided by it (should be `ad-1`, but could be different)